service/history/task/timer_active_task_executor.go

```diff
@@ -203,6 +203,30 @@ Loop:
 			return err
 		}
 		updateMutableState = true
+
+		if t.config.EnableDebugMode && t.config.EnableTimerDebugLogByDomainID(task.DomainID) {
+			t.logger.Info("User timer fired",
```

Comment:
nit: this is info log but name of the config says debug

Reply:
This is my intention